src/app/components/modules/Header/index.jsx

```diff
@@ -283,7 +283,18 @@ class Header extends React.Component {
                     )}
                     {/* If announcement is shown, ad will not render unless it's in a parent div! */}
                     <div>
-                        <GptAd type="Basic" slotName="top_nav" />
+                        {this.props.gptEnabled ? (
```

Comment:
This comment block details the two changes I made here. I felt it was important to explain this since it is usually not a good practice to use inline styles.

Reply:
Having the height set to the final 90px height fixes the issue, however, if the ad server decides there is no ad to display or if ads are disabled in the Condenser config, we are stuck with a large header and a large empty space.

Around line 34
```this.gptadshown = event => {
            // This makes sure that the sticky header doesn't overlap the welcome splash.
            this.forceUpdate();
        };```
Maybe here, you can add a timeout of few seconds, then check if the DFP iframe has loaded or contains anything? Or maybe the 'slotRenderEnded' from googletag.pubads() might give some details on whether the content is empty or not